Build/Shared.Server.Extensions.targets

```diff
@@ -22,26 +22,32 @@
   <Target Name="ArtifactBuild_Configuration">
     <PropertyGroup>
       <Platforms>x64</Platforms>
-      <DatabaseSystems>SqlServer2012</DatabaseSystems>
-      <Browsers></Browsers>
+      <DatabaseSystems>SqlServer2012+NoDB</DatabaseSystems>
+      <Browsers>NoBrowser</Browsers>
+      <ExecutionRuntimes>Win_NET48</ExecutionRuntimes>
+      <TargetRuntimes>NET48</TargetRuntimes>
       <SkipDocumentation>True</SkipDocumentation>
     </PropertyGroup>
   </Target>
 
   <Target Name="CIProductionBuild_Configuration">
     <PropertyGroup>
       <Platforms>x64</Platforms>
-      <DatabaseSystems>SqlServer2012</DatabaseSystems>
-      <Browsers>Chrome</Browsers>
+      <DatabaseSystems>SqlServer2012+NoDB</DatabaseSystems>
+      <Browsers>Chrome+NoBrowser</Browsers>
+      <ExecutionRuntimes>Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
+      <TargetRuntimes>NET48</TargetRuntimes>
       <SkipDocumentation>True</SkipDocumentation>
     </PropertyGroup>
   </Target>
 
   <Target Name="FullBuildReduced_Configuration">
     <PropertyGroup>
       <Platforms>x64</Platforms>
-      <DatabaseSystems>SqlServer2012</DatabaseSystems>
-      <Browsers>Chrome+InternetExplorer</Browsers>
+      <DatabaseSystems>SqlServer2012+NoDB</DatabaseSystems>
+      <Browsers>Chrome+InternetExplorer+Firefox+Edge+NoBrowser</Browsers>

```

Comment:
Why do we test all browser versions in fullbuildreduced?

Reply:
FullBuildReduced is used for our git nightly builds, as I recall it was mainly there to run tests in all browsers in the first place